shared/domains/chelonia/localSelectors.js

```diff
@@ -90,5 +91,23 @@ export default (sbp('sbp/selectors/register', {
         sbp('okTurtles.events/emit', CONTRACTS_MODIFIED_READY, subscriptionSet, { added, removed })
       })
     })
+  },
+  'chelonia/externalStateWait': async function (contractID) {
```

Comment:
Above line 95 could you please give a detailed description of:

1. What this function is doing
2. Why it exists, e.g. the sorts of scenarios it should be used in